### ISSUE TITLE
Choos README

### ISSUE THREAD

**Issue by drym3r** [gogits/gogs#2272]
Hi. It would be cool to be capable to choose between differente README files, if the repository has more than one. For example, having README and README.md and choosing the second one, since for default chooses the one without extension.

**Comment by Unknwon** [comment]
Thanks your feedback!

> For example, having README and README.md and choosing the second one, since for default chooses the one without extension.

Can you rephrase this sentence?

**Comment by drym3r** [comment]
Sure. Imagine, you have two readme files. One is called readme.md and the other readme. By default, gogs uses the one calleds readme. It would be nice if I could choose the one called readme.md. 
Better this way?

**Comment by Unknwon** [comment]
Thanks for clarifying!

**Comment by bkcsoft** [comment]
Only semi-related, but it would be nice to show all readme-files one after the other. E.g. I have a Repo with `readme.md` (basically 'about this project') , `readme.build.md`(build instructions) and `readme.install.md`(installation instructions). Would be nice to show all three :)

IMO the order would be readme.md, readme, readme.* (in whatever order) (assuming they exist of course)

**Comment by drym3r** [comment]
Well, I wouldn't agree on that because in the case I said my readme files are all the same.

**Comment by bkcsoft** [comment]
@drym3r if both are the same, why does it matter which one it shows?

**Comment by drym3r** [comment]
@bkcsoft Because they are in different formats. One in md, for git repos, and the other in rst, for pypi. (in my case). Since gogits does not render rst and pypi does not suprenderport md, I have to have both of them.

**Comment by bkcsoft** [comment]
@drym3r well, if gogs can't render the .rst-file, it seems like a non-issue really ^.^ but I get your point.
I'd like it to be a settings that takes a list "readme.md,readme.build.md,[etc]", something like this (only a mock, havn't written any code yet)
<img width="960" alt="screenshot 9" src="https://cloud.githubusercontent.com/assets/4726179/12295649/b7dcb620-ba02-11e5-9f9d-f2e29dc541a1.png">

**Comment by drym3r** [comment]
The thing its that it tries to render it, if the name is 'README'. But it just shows the raw file. 
Btw, that mock would be awesome, since the user can choose which is the main readme x)

**Comment by Dasoren** [comment]
If this could support repo paths as well that would be cool, like "/Docs/README.md" vs just the "/README.md" file. This way I could store all .md files in the Docs folder and have the README file in the root at a CLI file and not a markdown file. Organization if the files is why I would like this option. :)

**Comment by Unknwon** [comment]
I think setting which one to render is good enough? It seems that other priority isn't matter at all?

**Comment by strk** [comment]
Given Gogs is trying to mimic github it might also be a good idea to default to README.md, README as the initial order.

**Comment by Fastidious** [comment]
I have found something the is affected by this. If there is a ` README-zh.md`, and a `README.md`, the first will take precedence, which isn’t the intended behaviour. Is there a way to fix that?
